CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# How to Contribute
+
+First please go through the [Community Contributions at Xwiki](https://dev.xwiki.org/xwiki/bin/view/Community/Contributing)
+
+# Raising an issue:
+ This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests submitting pull requests as well.
+ Please report issues here [Issues - xwiki-contrib/android-authenticator](https://jira.xwiki.org/projects/ANDAUTH/issues)
+
+# Branch Policy
+
+# Sending pull requests:
+
+* Go to the repository on github at https://github.com/xwiki-contrib/android-authenticator
+
+* Click the “Fork” button at the top right.
+
+* You’ll now have your own copy of the original Android-Authenticator repository in your github account.
+
+* Open a terminal/shell.
+
+* Type
+
+`$ git clone https://github.com/xwiki-contrib/android-authenticator.git`
+
+where 'username' is your github username.
+
+You’ll now have a local copy of your version of the original Xwiki repository.
+
+# Change into that project directory (android-authenticator):
```

Comment:
It is look like a part of previous subsection. I think, it would be better to replace it above